src/windows/lswclient/DllMain.cpp

```diff
@@ -0,0 +1,283 @@
+/*++
+
+Copyright (c) Microsoft. All rights reserved.
+
+Module Name:
+
+    DllMain.cpp
+
+Abstract:
+
+    This file the entrypoint for the LSW client library.
```

Comment:
I'm still not convinced we need the DLL. We already have a COM interface, why not just use that?